<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeSlot</class>
 <widget class="QWidget" name="ChargeSlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>582</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="laySettings">
     <item>
      <widget class="QGroupBox" name="grpChargeParam">
       <property name="title">
        <string>Charging parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btnSettings">
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblBatType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Battery type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lstBatteryType"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lauNbCells">
          <item>
           <widget class="QLabel" name="lblCells">
            <property name="text">
             <string>Cells :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblNCells">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="sldCells">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblChargeMode">
          <property name="text">
           <string>Charging mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lstChargingMode"/>
        </item>
        <item>
         <widget class="QLabel" name="lblChargeCurrent">
          <property name="text">
           <string>Charge current:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spnChargeCurrent">
          <property name="suffix">
           <string>A</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDischargeCurrent">
          <property name="text">
           <string>Discharge current:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spnDischargeCurrent">
          <property name="suffix">
           <string>A</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>2.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblRepeakCount">
          <property name="text">
           <string>Repeak count:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spnRepeakCount">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCutVoltage">
          <property name="text">
           <string>Cut voltage :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spnCutVoltage">
          <property name="suffix">
           <string>V</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>25.199999999999999</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>3.200000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblNbCycle">
          <property name="text">
           <string>Cycle count:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spnCycleCount">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCycleOrder">
          <property name="text">
           <string>Cycle order:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lstCycleOrder"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSurveiller">
       <property name="text">
        <string>Monitor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layGraph">
     <item>
      <widget class="QLabel" name="lblMessage">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QChartView" name="chartView1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>40</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chkTemperature">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Temperature</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkVoltage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Voltage</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCurrent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Current</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCapacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Capacity</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layInfo">
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblCell1">
       <property name="text">
        <string>Cell 1: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCell2">
       <property name="text">
        <string>Cell 2: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCell3">
       <property name="text">
        <string>Cell 3:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCell4">
       <property name="text">
        <string>Cell 4:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCell5">
       <property name="text">
        <string>Cell 5:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCell6">
       <property name="text">
        <string>Cell 6:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblCurTime">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCurVoltage">
       <property name="text">
        <string>Voltage:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCurCurrent">
       <property name="text">
        <string>Current:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCurCapacity">
       <property name="text">
        <string>Capacity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCurTemp">
       <property name="text">
        <string>Temperature:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
